### ISSUE TITLE
GitHub mentions aren't themed

### ISSUE THREAD

**Issue by Insprill** [catppuccin/userstyles#512]
### Is there an existing issue outlining your problem?

- [X] I have searched the existing issues and they do not solve my problem.

### What userstyles are you seeing the problem on?

lbl:github

### Describe your problem.

The background of mentions doesn't get themed.

### Attach screenshots.

![image](https://github.com/catppuccin/userstyles/assets/52843537/65c4641c-7ee7-4448-b29a-07f0654c9d4f)

### What browser(s) are you seeing the problem on?

Firefox v122.0.1

**Comment by uncenter** [comment]
Sorry @isabelroses... just for testing.

**Comment by uncenter** [comment]
Yeah okay just as I had thought. They are themed, but I believe your accent color just happens to look like that and makes you think it isn't themed?

I use the `red` accent color:
![Screenshot 2024-02-13 at 11 11 43 (Arc)](https://github.com/catppuccin/userstyles/assets/47499684/69be4758-3ebb-4cb7-bed2-243a32a9f5a5)
But with `mauve`:
![Screenshot 2024-02-13 at 11 12 49 (Arc)](https://github.com/catppuccin/userstyles/assets/47499684/1fdf6d84-73a0-4655-ad34-22d8e2602225)

**Comment by isabelroses** [comment]
> Yeah okay just as I had thought. They are themed, but I believe your accent color just happens to look like that and makes you think it isn't themed?
> 
> I use the `red` accent color: ![Screenshot 2024-02-13 at 11 11 43 (Arc)](https://private-user-images.githubusercontent.com/47499684/304476227-69be4758-3ebb-4cb7-bed2-243a32a9f5a5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc4NDEzMDYsIm5iZiI6MTcwNzg0MTAwNiwicGF0aCI6Ii80NzQ5OTY4NC8zMDQ0NzYyMjctNjliZTQ3NTgtM2ViYi00Y2I3LWJlZDItMjQzYTMyYTlmNWE1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjEzVDE2MTY0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU1YzNiZjNhZDVjMTQ2NTFiMDBlMTE2YmUxM2ExYWIyM2Q5NTA1ZjFmYjAwM2M0NmY1ZDU1ZDkxMGIyZWYwZmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.56NPm3g_p3lg2OV452ZQniFgNaxM2Zkcc9FoaGLe0Jw) But with `mauve`, it looks like the color in your screenshot: ![Screenshot 2024-02-13 at 11 12 49 (Arc)](https://private-user-images.githubusercontent.com/47499684/304476551-1fdf6d84-73a0-4655-ad34-22d8e2602225.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc4NDEzMDYsIm5iZiI6MTcwNzg0MTAwNiwicGF0aCI6Ii80NzQ5OTY4NC8zMDQ0NzY1NTEtMWZkZjZkODQtNzNhMC00NjU1LWFkMzQtMjJkOGUyNjAyMjI1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjEzVDE2MTY0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc4MGM0NmM5ZWY2N2M1YjlkMzJhZTYzOTcxNWNiOGYwMzBmMzBjYjRlNWM0ZDFhNjljNThlZWExNjI4MWRhMWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Ye1itI7WqL6-NFTrXnEJEBjrdZw8xSdbWI1PbFA4-FA)

![image](https://github.com/catppuccin/userstyles/assets/71222764/41af797a-313f-474d-8b0c-5cff96204cb3)

**Comment by uncenter** [comment]
Huh, there aren't backgrounds on mentions for me...

**Comment by uncenter** [comment]
Turns out you have to be pinged by someone else to see it, since the background is a highlight that only appears when you yourself are mentioned.
